terraform/projects/meom-ige.tfvars

```diff
@@ -0,0 +1,44 @@
+prefix     = "meom-ige"
+project_id = "meom-ige-cnrs"
+
+# Minimum number of nodes required to fit kube-system is either
+# 2 n1-highcpu-2 nodes, or 3 g1-small nodes. If you don't enable
+# networkpolicy, you can get away with 1 n1-custom-4-3840 node -
+# but with that enable, calico-typha wants 2 replicas that
+# must run on two nodes since they both want the same hostport.
+# 3 g1-small is 13$ a month, wile a single n2-highcpu-2 is
+# already 36$ a month. We want very low base price, and
+# our core nodes will barely see any CPU usage, so g1-small is
+# the way to go
+core_node_machine_type = "g1-small"
+
+enable_network_policy    = false
+config_connector_enabled = false
+
+notebook_nodes = {
+  "small" : {
+    min : 0,
+    max : 20,
+    machine_type : "e2-standard-2"
+  },
+  "medium" : {
+    min : 0,
+    max : 20,
+    machine_type : "e2-standard-8"
+  },
+  "large" : {
+    min : 0,
+    max : 20,
+    machine_type : "e2-standard-16"
+  },
+  "very-large" : {
+    min : 0,
+    max : 20,
+    machine_type : "e2-standard-32"
+  }
+}
```

Comment:
I see the versatility to have this defined per cluster... I also maybe see some upcoming duplication among GKE clusters? Wondering if a "sensible" default may help with that... actually not sure about that neither.

Reply:
Around different kind of instance sizes?

Reply:
In your last commit, you added the terraform/projects/cloudbank.tfvars and it looks pretty similar to the https://github.com/2i2c-org/pilot-hubs/blob/master/terraform/pilot-hubs.tfvars, so I am wondering if we should have (or not) a "sensible" default representing that sort of "common" cluster.

Reply:
@damianavila hmm, maybe - can we open it up as another issue? I don't want to keep this PR open for too long. I don't fully have a sense of how to cascade configuration with tfvars. And since we only have two that are so common, I don't mind this right now.

Reply:
> can we open it up as another issue? 

Totally OK with that!